examples/insulin-tutorial/reports/scaling_model.ipynb

```
@@ -0,0 +1,464 @@
+    "from mdx2.io import loadobj\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c497ca1e",
+   "metadata": {
+    "papermill": {
+     "duration": 0.001433,
+     "end_time": "2026-04-17T17:35:11.555888+00:00",
+     "exception": false,
+     "start_time": "2026-04-17T17:35:11.554455+00:00",
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)